x-pack/plugin/src/test/resources/rest-api-spec/test/ml/update_model_snapshot.yml

```diff
@@ -1,207 +1,3 @@
-        Content-Type: application/json
-      index:
-        index:  .ml-anomalies-update-model-snapshot
-        id:     "update-model-snapshot_model_snapshot_snapshot-1"
-        body: >
-          {
-            "job_id" : "update-model-snapshot",
-            "timestamp": "2016-06-02T00:00:00Z",
-            "snapshot_id": "snapshot-1",
-            "snapshot_doc_count": 3,
-            "retain": false
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-      index:
-        index:  .ml-state-000001
-        id:     "update-model-snapshot_model_state_1#1"
-        body:  >
-          {
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-      index:
-        index:  .ml-state-000001
-        id:     "update-model-snapshot_model_state_1#2"
-        body:  >
-          {
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-      index:
-        index:  .ml-state-000001
-        id:     "update-model-snapshot_model_state_1#3"
-        body:  >
-          {
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-        Content-Type: application/json
-      index:
-        index:  .ml-anomalies-update-model-snapshot
-        id:     "update-model-snapshot_model_snapshot_snapshot-2"
-        body: >
-          {
-            "job_id": "update-model-snapshot",
-            "timestamp": "2016-06-01T00:00:00Z",
-            "snapshot_id": "snapshot-2",
-            "description": "snapshot 2 description",
-            "snapshot_doc_count": 2,
-            "retain": true
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-      index:
-        index:  .ml-state-000001
-        id:     "update-model-snapshot_model_state_2#1"
-        body:  >
-          {
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-      index:
-        index:  .ml-state-000001
-        id:     "update-model-snapshot_model_state_2#2"
-        body:  >
-          {
-          }
-
-  - do:
-      headers:
-        Authorization: "Basic eF9wYWNrX3Jlc3RfdXNlcjp4LXBhY2stdGVzdC1wYXNzd29yZA==" # run as x_pack_rest_user, i.e. the test setup superuser
-      indices.refresh:
-        index: [.ml-anomalies-update-model-snapshot,.ml-state*]
-
----
-"Test with valid description":
```

Comment:
The blacklist in `x-pack/plugin/ml/qa/ml-with-security/build.gradle` contains an entry for `ml/update_model_snapshot/Test without description`.  It looks like this had already been deleted, but since you're editing both files please can you delete it from the blacklist.

Reply:
yeah totally spaced on the security test black lists 🤦  will take care of that